konnectivity-client/pkg/common/metrics/testing/metrics.go

```diff
@@ -30,6 +30,11 @@ const (
 # HELP konnectivity_network_proxy_client_dial_failure_total Number of dial failures observed, by reason (example: remote endpoint error)
 # TYPE konnectivity_network_proxy_client_dial_failure_total counter`
 	clientDialFailureSample = `konnectivity_network_proxy_client_dial_failure_total{reason="%s"} %d`
+
+	tunnelsHeader = `
+# HELP konnectivity_network_proxy_client_tunnels Number of in-use tunnels, by status (Example: dialing)
```

Comment:
I don't believe tunnels is a well defined term. To me the tunnel is the OS visible network connection over which individual connections are transported (tunneled). However it seems other folks have other ideas. I think it would be clearer to phrase things in terms on GRPC/HTTP-Connect connections and client/request connections.

Reply:
I believe this will become a lot more important if we write the code to allow the konnectivity client to mutex the client connections over a single GRPC connection, the way the server and agent currently do.

Reply:
client_connections seems good for the request level connections.

Still need something for the higher level connection.